tests/unit/CardUtilsTest.ts

```diff
@@ -272,4 +311,23 @@ describe('CardUtils', () => {
             expect(feedName).toBe('');
         });
     });
+
+    describe('getFilteredCardList', () => {
+        it('Should return filtered custom feed cards list', () => {
+            const cardsList = CardUtils.getFilteredCardList(customFeedCardsList, undefined);
+            // eslint-disable-next-line @typescript-eslint/naming-convention
+            expect(cardsList).toStrictEqual({'480801XXXXXX2111': 'ENCRYPTED_CARD_NUMBER'});
+        });
```

Comment:
Can you add a test for more than one card?

Reply:
@mountiny Updated!